### ISSUE TITLE
508-defect-2 [COGNITION]: Remove buttons SHOULD have accessible labels when editing inline data on Review screen

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#12999]
## [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The Remove buttons for removing dependents on the Review > edit line UI need `aria-label` so screen reader users understand what they are removing without having to navigate back to the nearest heading. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] As a screen reader user, I want to hear an accessible label like "Remove former spouse" instead of "Remove" when the button has focus or virtual cursor.

## Environment
* All screen readers
* All browsers

## Steps to Recreate

* Turn on a screen reader of your choice
* Enter https://staging.va.gov/view-change-dependents/add-remove-form-686c/ in browser
* Check two or three dependent situations, including adding and removing at least one dependent each
* Navigate to the **Review** page and open the accordions
* Click "Edit" to enter inline editing mode, add a dependent, then click the "Remove" button after the generic card appears to add a dependent.
* Verify you only hear "Remove" and have to navigate to the heading to answer "Remove what?"

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-08-28 at 9 46 06 AM](https://user-images.githubusercontent.com/934879/91612938-18b84980-e944-11ea-9f43-d35987368271.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by jason-gcio** [comment]
Out of the (Form System) box, the tags for the Remove button are not dynamic - we will not be able to get into the level that would allow us to make this change.  Has any other team resolved this at their own application level?

**Comment by jason-gcio** [comment]
Confirm but we may not be able to change this

**Comment by micahchiang** [comment]
@jason-gcio I believe this ticket can be closed. As seen in the screenshot below, aria-labels have been added to remove buttons since the creation of this ticket.

<img width="463" alt="Screen Shot 2021-08-18 at 10 57 57 AM" src="https://user-images.githubusercontent.com/15097156/129931497-c3327002-38cc-47b1-a76f-afda621a175f.png">
